ludwig/schema/llms/peft.py

```diff
@@ -101,6 +106,18 @@ class LoraConfig(BaseAdapterConfig):
         description="Bias type for Lora.",
     )
 
+    target_modules: Optional[List[str]] = schema_utils.List(
+        default=None,
+        allow_none=True,
+        description=(
+            "List of module names or regex expression of the module names to replace with LoRA. "
+            "For example, ['q', 'v'] or '.*decoder.*(SelfAttention|EncDecAttention).*(q|v)$'. "
+            "Defaults to targeting the query and value matrices of all self-attention and encoder-decoder attention "
```

Comment:
@tgaddair Just for my understanding: where is this default specified?  Is it inherent in the Transformers/PEFT implementation?  Thank you.

Reply:
Yes, it's deep in the weeds of the PEFT code base:

https://github.com/huggingface/peft/blob/main/src/peft/utils/other.py#L481